about.html

```diff
@@ -84,27 +92,28 @@ <h4 class="about-title-details">Lorem ipsum dolor sit amet consectetur, adipisic
   <section class="footer">
     <section class="partner">
       <div class="partner-title">
-        <img src="image/partner-header-img.png" alt="">
+        <img src="image/partner-header-img.png" alt="image">
       </div>
       <div class="client">
         <ul class="client-ul">
-          <img src="image/partner-mozila.png" alt="">
-          <img src="image/partner-google.png" alt="">
-          <img src="image/partner-navbar.png" alt="">
-          <img src="image/partner-daumk.png" alt="">
-          <img src="image/partner-airbnb.png" alt="">
+          <img src="image/partner-mozila.png" alt="image">
+          <img src="image/partner-google.png" alt="image">
+          <img src="image/partner-navbar.png" alt="image">
+          <img src="image/partner-daumk.png" alt="image">
+          <img src="image/partner-airbnb.png" alt="image">
         </ul>
       </div>
     </section>
     <section class="footerLast">
       <div class="footer-logo">
-        <img src="image/footer-img.png" alt="">
+        <img src="image/footer-img.png" alt="image">
       </div>
       <div class="footer-description">
         <h5 class="footer-title">2015 Creative Commons Korea.</h5>
-        <h6 class="footer-title2">Lorem ipsum dolor sit, amet</h6>
+        <h6 class="footer-title2">Lorem ipsum dolors sit, amet.</h6>
       </div>
     </section>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the footer in the `About` page has a dark background so it looks exactly as the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230325_153221](https://user-images.githubusercontent.com/25227022/227727043-b933a0c7-208d-47e8-b526-6269e94b7fc2.png)|![Screenshot_20230325_153625](https://user-images.githubusercontent.com/25227022/227727130-2b2f9718-638a-47b4-826f-b469d48e983c.png)|